changelogs/fragments/56357-ibm-storage-unwanted-args-cause-failure.yml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+- ibm_storage - Added a check for null fields in ibm_storage utils module.
```

Comment:
Note, it's better if changelogs reflect what was changed from the POV of the user.  For instance, fixed a traceback when storage options were unset in the initial configuration.  (I don't know if that's actually what is happening here, it's just an example ;-)